### DISCUSSION TITLE
"Charterzug" not showing up in Träwelling

### DISCUSSION THREAD

**Discussion by Quadrubo** [Traewelling/traewelling#3146] [General]
### Describe the bug

Today I will be traveling with a special train by the "Berliner Eisenbahnfreunde e.V.". I did not expect to find this train on [bahn.expert](https://bahn.expert/details/DZ%2083272/?journeyId=20241208-933d6795-a0f3-315a-90de-5a92f93becde) but I did.
Then as I wanted to check in, I couldn't find the train in Träwelling.

![image](https://github.com/user-attachments/assets/f5bbf549-dbc7-4641-bdf6-1782e31d972e)

![image](https://github.com/user-attachments/assets/d923c5c7-6d25-4f9f-b2e5-49bf36b51ab3)

If this is an issue, I would gladly help debugging it on future trips.

### Steps to reproduce

Try to check in from "Berlin-Lichtenberg". Look at the trips at 13:16.

**Comment by MrKrisKrisu** [comment]
As this is related to the old data provider we were using and it's out of our control, there's nothing we can do about it.

**Comment by d3d9** [comment]
can this be converted into a discussion and be kept open? imo this never was related to the old data provider, but more to a potential RIS data provider?
